i18n/en.pot

```diff
@@ -29,6 +29,9 @@ msgid ""
 "this dashboard?"
 msgstr ""
 
+msgid "Failed to save the dashboard"
```

Comment:
It would be useful to be more specific, or suggestive, of a possible fix. I'm thinking `Failed to save dashboard. You might be offline or not have access to edit this dashboard.` Are there other reasons this error might end up being displayed? Perhaps "does not exist anymore"?

Reply:
Does not exist could be possible, though probably a very unlikely edge case. It would have had to be deleted between the time the user opened edit mode, and clicked save.

I asked Maikel about different error codes from the backend and he said it was pretty complicated. So I like your idea of keeping it generic.

Reply:
OK, I think we can keep it generic with `Failed to save dashboard. You might be offline or not have access to edit this dashboard.`. The "might" is enough to suggest that these aren't the only possible reasons, but the most likely.